### DISCUSSION TITLE
ui.toggle set QBtnToggle Stack = True

### DISCUSSION THREAD

**Discussion by pingretispec** [zauberzeug/nicegui#1848] [Q&A] [answered]
### Question
Does anyone know how to use the "stack" content of a QBtnToggle?

Spread works like this:
```py
toggle = ui.toggle(options, clearable=True).props('spread')
```

But this doesn't work
```py
toggle = ui.toggle(options, clearable=True).props('stack')
```

**Comment by falkoschindler** [comment, marked as answer]
From the [QBtnToggle documentation](https://quasar.dev/vue-components/button-toggle#qbtntoggle-api) about the "stack" prop:
> Stack icon and label vertically instead of on same line (like it is by default)

Since `ui.toggle` currently doesn't support option icons, this prop has not effect.

In case you expect the options to be stacked vertically: I don't think QBtnToggle can do that. The options are always put in a row.

**Comment by pingretispec** [comment]
![image](https://github.com/zauberzeug/nicegui/assets/74611983/013b9f54-cae5-4ea3-9761-d0a1f4e49494)
Is this "stack" call only stacking icon with label? Not stacking toggles?

**Reply by rodja** [reply]
Yes, thats what the Quasar docs says.

**Comment by danilogalisteu** [comment]
For posterity, after messing around with dev tools, I was able to stack the toggle buttons and set a common width with:
```
ui.toggle(...).classes(add="column", remove="row no-wrap inline").style("width: 120px")
```
Using version 2.0.1.
